### ISSUE TITLE
/create route additional params

### ISSUE THREAD

**Issue by CodeF53** [howardchung/watchparty#858]
I would like to be able use the `/create?video=...` route with a couple of extra parameters to improve user experience:
1. `&subtitles=`
I was thinking this would take a json format
```json
{
  "en-us": "https://.../eng.vtt",
  "es": "https://.../spa-7.vtt"
}
```
2. `&title=`
Simply so the "search by title" option for subtitles works with parties started this way
3. `&poster=` an image to pass to the `<video poster="...">` attribute (shows while video is downloading)

Full example url
```
https://www.watchparty.me/create
  ?video=https://.../playlist.m3u8
  &title="EXAMPLE TITLE"
  &subtitles={"en-us":"https://.../eng.vtt","es":"https://.../spa-7.vtt"}
  &poster=https://.../poster.jpg
```

**Comment by howardchung** [comment]
I don't think we currently have support for multiple subtitle tracks, but everything else sounds good!

**Comment by CodeF53** [comment]
I'm fine just passing in a single subtitle track if it would be difficult to implement support for multiple tracks
